models/_generate_layers.py

```diff
@@ -496,6 +497,46 @@ def collect_layers():
   return ls
 
 
+def get_super_call_kwargs_repr(layer: LayerBase) -> str:
+  """
+  Inspects the given layer in RETURNN, extracts the super call from there, removes unwanted parameters.
+
+  :param layer: Layer to be inspected
+  :return: A string containing all arguments of the super() call of the layer which are not excluded.
+           This then can directly be used when writing the super call for the layer for the _generated_layers file
```

Comment:
This is still not clear.
I would write sth like:
```suggestion
  :return: kwargs_repr
    Python source code of kwargs for the super() call of the layer which are not excluded.
    Such that `f"super().__init__({kwargs_repr})"` is valid code.
```